main.c

```diff
@@ -332,6 +332,12 @@ static void usage(const char *argv0)
         "caps except CAP_NET_BIND_SERVICE if running as the root)\n");
     printf("--enable-seccomp         enable seccomp to limit syscalls "
            "(experimental)\n");
+    printf("--outbound-addr=IPv4     sets outbound ipv4 address to bound to "
```

Comment:
Add `/* v1.1.0 */` comment (see above lines)

Reply:
Also please update the man page

Reply:
done